frontend/src/global.css

```
@@ -0,0 +1,42 @@
+  --color-border-header: #EAECEF;
+
+  /* button color */
+  --color-btn-primary-bg: #0F172A;
+  --color-btn-primary-bg-hover: #1E293B;
+  --color-btn-primary-text: #ffffff;
+  --color-btn-secondary-bg: #E2E8F0;
+  --color-btn-secondary-bg-hover: #CBD5E1;
+  --color-btn-secondary-text: #475569;
+
+  /* fonts */
+  --font-family-base: Inter, -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.9.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)